src/main/java/lotto/model/WinningNumbers.java

```diff
@@ -0,0 +1,68 @@
+package lotto.model;
+
+import lotto.exception.InvalidInputException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static lotto.common.Constants.*;
+
+public class WinningNumbers {
+
+    private static final String LIST_SEPARATOR = ",";
+    private static final String INVALID_NUMBER_COUNT_MESSAGE= "당첨번호 %s개를 입력해야 합니다.";
+    private static final String INVALID_DUPLICATE_MESSAGE= "당첨번호를 중복입력을 할 수 없습니다.";
+    private static final String INVALID_WINNING_NUMBER_MESSAGE= "%s-%s 사이의 숫자만 입력할 수 있습니다.";
+
+    private List<Integer> values;
```

Comment:
객체지향 생활 체조 원칙
규칙 3: 모든 원시값과 문자열을 포장한다.

```suggestion
    private List<LottoNumber> values;
```

`클래스 이름은 예시 일 뿐입니다.`

이런식으로 원시값 랩핑을 해보면 좋을 것 같아요 :)

적절한 validation 로직이나 책임을 가질 수 있다면 특정 객체로 랩핑하는것이 유의미하다고 생각합니다 👍

Reply:
놓쳤던 부분인데, 피드백 고맙습니다.
생각해보니 WinningNumbers에서 원시값에 대한 Validation까지 같이 처리하고 있었습니다.
해당 부분 수정해보겠습니다.😀👍